### ISSUE TITLE
METADATA 2.0

### ISSUE THREAD

**Issue by a1694993165a** [IBMa/equal-access#1320]
### Rule description

Meamingful updates for the previous data infrastructure by having better time performa
[guide-for-peer-reviewers_he-erc-poc_en.pdf](https://github.com/IBMa/equal-access/files/10834519/guide-for-peer-reviewers_he-erc-poc_en.pdf)
ce 

### Relevant requirement(s)

4.1.4 Accessibility-supported Technologies Only (IBM requirement)

### Ruleset and failure level

WCAG 2.1

### Failure level

Potential violations, Recommendation

### Different failure level

[European Research Council he-erc-poc_en.pdf](https://github.com/IBMa/equal-access/files/10834526/European.Research.Council.he-erc-poc_en.pdf)

**Comment by tombrunet** [comment]
Will not open attachments. Looks like spam.
